yaus_c4gt_frontend/src/pages/SignIn.js

```diff
@@ -0,0 +1,323 @@
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { useHistory } from "react-router-dom";
+import {
+  Layout,
+  Menu,
+  Button,
+  Row,
+  Col,
+  Typography,
+  Form,
+  Input,
+  Switch,
+} from "antd";
+import signinbg from "../assets/images/img-signin.jpg";
+import {
+  DribbbleOutlined,
+  TwitterOutlined,
+  InstagramOutlined,
+  GithubOutlined,
+} from "@ant-design/icons";
+function onChange(checked) {
+  console.log(`switch to ${checked}`);
+}
+const { Title } = Typography;
+const { Header, Footer, Content } = Layout;
+const template = [
```

Comment:
This code is reduntant from the SignUp page

Reply:
resolved in b0702c3ac5bc997f4a75e0142614dda2c4d641ae